__tests__/components/editor/InputLiteral.test.js

```diff
@@ -90,5 +91,32 @@ describe('<InputLiteral />', () => {
     expect(wrapper.find('button#displayedItem').length).toEqual(2)
     wrapper.find('button#displayedItem').first().simulate('click', { target: { "dataset": {"item": 0 }}});
     expect(wrapper.find('button#displayedItem').length).toEqual(1)
+    wrapper.setState({content_add : '', myItems: []}) /** reset state **/
+
+  })
+  it('valueConstraint contains defaults but it is an empty array', () => {
+    wrapper.instance().defaultLiteralValue()
+    wrapper.instance().forceUpdate()
+    console.log(wrapper.html())
```

Comment:
Console statement should be removed from test :-)

Reply:
i always forget to remove one :(